Documentation/Tutorials/EdgeTutorialAppFinal/Podfile

```diff
@@ -8,10 +8,10 @@ target 'EdgeTutorialApp' do
 #=begin
   pod 'AEPAssurance'
   pod 'AEPCore'
-  pod 'AEPEdge'
-  pod 'AEPEdgeConsent'
-  pod 'AEPEdgeIdentity'
   pod 'AEPLifecycle'
   pod 'AEPServices'
+  pod 'AEPEdge', :git => 'https://github.com/adobe/aepsdk-edge-ios.git', :branch => 'dev'
```

Comment:
Should we revert these back to production pod?

Reply:
After release